src/Symfony/Bundle/FrameworkBundle/Resources/config/cache_pools.xml

```diff
@@ -13,11 +13,7 @@
         <service id="cache.adapter.shared" alias="cache.adapter.filesystem" />
         <service id="cache.adapter.local" alias="cache.adapter.filesystem" />
 
-        <service id="cache.pool.shared" parent="cache.adapter.shared">
-            <tag name="cache.pool" clearer="cache.default_pools_clearer" />
-        </service>
-
-        <service id="cache.pool.local" parent="cache.adapter.local">
+        <service id="cache.pool.app" parent="cache.adapter.shared">
```

Comment:
@javiereguiluz WDYT about this one? We'd like to provide a preconfigured cache pool for user's need, so that one can inject it without much configuration. Currently we have two of these (`cache.pool.local` and `cache.pool.shared`).
The proposal here is to remove `cache.pool.local` and rename `cache.pool.shared` to `cache.pool.app`.
Which means we have to questions:
- OK for removing the local pool and keep only the shared one?
- is the `cache.pool.app` name OK or do we have a better idea (e.g. is `cache.pool.user` better)?